src/Symfony/Component/HttpFoundation/InputBag.php

```diff
@@ -73,6 +73,14 @@ public function set(string $key, mixed $value)
         $this->parameters[$key] = $value;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getString(string $key, string $default = ''): string
```

Comment:
Shortcuts the parent method because the validation on scalar is already done un `get()`.